key-rotator/manifest/manifest.go

```diff
@@ -352,7 +357,7 @@ type BatchSigningPublicKey struct {
 	// PublicKey is the PEM armored base64 encoding of the ASN.1 encoding of the
 	// PKIX SubjectPublicKeyInfo structure. It must be an ECDSA P256 key.
 	PublicKey string `json:"public-key"`
-	// Expiration is the ISO 8601 encoded UTC date at which this key expires.
+	// Expiration is the RFC3339-encoded UTC date at which this key expires.
```

Comment:
hair-splitting-nit: [in the design doc](https://docs.google.com/document/d/1MdfM3QT63ISU70l63bwzTrxr93Z7Tv7EDjLfammzo6Q/edit#bookmark=kix.yx44j8g5efpv), we do define these dates as ISO 8601, which I chose because [W3C says it's the "right" JSON date format](https://www.w3docs.com/snippets/javascript/the-right-json-date-format.html). AFAICT, Go's `time.RFC3339` is a valid ISO 8601 date, since it doesn't use a signed offset, which [Wikipedia says is the one way RFC 3339 deviates from ISO 8601](https://en.wikipedia.org/wiki/ISO_8601#RFCs).

So I think documenting that we use ISO 8601 is most correct, because that's the contract with other servers. That we use Go's `time.RFC3339` is an implementation detail.